src/main/java/teammates/logic/api/Logic.java

```diff
@@ -1091,6 +1091,13 @@ public void putDocument(StudentAttributes student) {
     }
 
     /**
+     * Return error message if no instructor display to students.
```

Comment:
Return**s {@code true}** if **at least one of the instructors is visible to students**